ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -689,85 +699,110 @@
+            [leaflet/Map {:ref "leaflet"
+                          :center #js [65 25]
+                          :zoomControl true
+                          :zoom 5}
+             (leaflet/background-tile-map)
+             (when show-date1?
+               ^{:key (str date1 "_" route-short-name "_" route-long-name "_" zoom)}
+               [leaflet/GeoJSON {:data date1-data
+                                 :onEachFeature (initialize-route-features (- offset) icon-size)
+                                 :style {:lineJoin "miter"
+                                         :lineCap "miter"
+                                         :color style/date1-highlight-color
+                                         :weight line-weight}}])
+             (when show-date2?
+               ^{:key (str date2 "_" route-short-name "_" route-long-name "_" zoom)}
+               [leaflet/GeoJSON {:data date2-data
+                                 :onEachFeature (initialize-route-features (+ 2 offset) icon-size)
+                                 :style {:lineJoin "miter"
+                                         :lineCap "miter"
+                                         :color style/date2-highlight-color
+                                         :weight line-weight}}])]]))})))
 
 (defn selected-route-map-section [e! open-sections date->hash hash->color compare]
   [section {:toggle! #(e! (tv/->ToggleSection :route-map))
             :open? (get open-sections :route-map true)}
    "Kartta"
-   [:span
-    "Kartalla näytetään valitsemasi reitin ja päivämäärien mukainen pysäkkiketju sekä ajoreitti, mikäli se on saatavilla."
+   [:div
+    "Valitse kartalla näytettävät pysäkkiketjut. Alla olevaan listaan on koostettu kaikki erilaiset pysäkkiketjut valitsemasi reitin ja kalenterin päivien vuorojen perusteella."
     [:div (stylefy/use-style {:margin-top "1.25rem"
-                              :margin-bottom "1.25rem"})
-     [ui/checkbox {:label "Näytä pysäkit"
-                   :checked (boolean (:show-stops? compare))
-                   :on-check #(e! (tv/->ToggleRouteDisplayStops))}]]
-    (when (> (count (:show-route-lines compare)) 1)
-      ;; There is more than one distinct route (stop-sequence), show checkboxes for displaying
-      (doall
-       (for [[routename show?] (sort-by first (seq (:show-route-lines compare)))]
-         ^{:key routename}
-         [ui/checkbox {:label routename
-                       :checked show?
-                       :on-check #(e! (tv/->ToggleShowRouteLine routename))}])))]
-   [selected-route-map e! date->hash hash->color compare]])
+                              :margin-bottom "1.25rem"
+                              :display "flex"
+                              :flex-direction "row-reverse"
+                              :flex-wrap "nowrap"
+                              :justify-content "space-between"})
+
+     [:div {:style {:width "20%"}}
```

Comment:
Was it tested if the :width percentages work on narrow screens like mobile?

Reply:
I didn't test with mobile screens. This page won't work in narrow screens, but I did take flex into use in this. Thank you for pointing this out.